nui/src/main/java/org/terasology/nui/Color.java

```diff
@@ -41,19 +43,41 @@
  *
  */
 @API
-public class Color {
+public class Color implements Colorc{
 
+    @Deprecated
     public static final Color BLACK = new Color(0x000000FF);
+    @Deprecated
     public static final Color WHITE = new Color(0xFFFFFFFF);
+    @Deprecated
     public static final Color BLUE = new Color(0x0000FFFF);
+    @Deprecated
     public static final Color GREEN = new Color(0x00FF00FF);
+    @Deprecated
     public static final Color RED = new Color(0xFF0000FF);
+    @Deprecated
     public static final Color GREY = new Color(0x888888FF);
+    @Deprecated
     public static final Color TRANSPARENT = new Color(0x00000000);
+    @Deprecated
     public static final Color YELLOW = new Color(0xFFFF00FF);
+    @Deprecated
     public static final Color CYAN = new Color(0x00FFFFFF);
+    @Deprecated
     public static final Color MAGENTA = new Color(0xFF00FFFF);
 
+    public static final Colorc black = new Color(0x000000FF);
```

Comment:
Why do we want to expose only `Colorc` instead of `Color`?  Should this rather be a helper class for working with JOML's color?

Reply:
Ah, now I got that `Colorc` is our own doing 🤓 (similar to what JOML does) 💡